Source/WebCore/PAL/pal/graphics/WebGPU/Impl/WebGPUSurfaceImpl.cpp

```diff
@@ -49,7 +49,11 @@ SurfaceImpl::~SurfaceImpl()
 
 void SurfaceImpl::destroy()
 {
+    if (!m_backing)
```

Comment:
these changes are from https://github.com/WebKit/WebKit/pull/8941

which is on the merge-queue so the WebCore changes will go away after I merge that and rebase